answerking_app/models/serializers.py

```diff
@@ -0,0 +1,157 @@
+import re
+
+from django.core.validators import (
+    MaxValueValidator,
+    MinValueValidator,
+    RegexValidator,
+)
+from rest_framework import serializers
+
+from answerking_app.models.models import Category, Item, Order, OrderLine
+
+MAXNUMBERSIZE = 2147483647
+
+
+def compress_white_spaces(value: str) -> str:
+    return re.sub(" +", " ", value)
+
+
+class ItemSerializer(serializers.ModelSerializer):
+    name = serializers.CharField(
+        max_length=50,
+        allow_blank=False,
+        validators=[RegexValidator("^[a-zA-Z !]+$")],
+        trim_whitespace=True,
+    )
+    price = serializers.DecimalField(
+        max_digits=19,
```

Comment:
seems large, maybe 9? that way max price is 100,000.00?

Reply:
well, I haven't decided that, it was the default value size that was decided during the Academy

Reply:
ok leave for now